pkg/component/observability/plutono/plutono_test.go

```diff
@@ -697,6 +709,37 @@ status:
 					})
 				})
 
+				Context("managed by gardener-operator", func() {
+					BeforeEach(func() {
+						values.OnlyDeployDataSourcesAndDashboards = true
+					})
+
+					It("should successfully deploy all resources", func() {
+
+						dashboardConfigMapName := "plutono-dashboards-garden"
+						dashboardCount := 28
+						GinkgoHelper()
```

Comment:
AFAIK, `GinkgoHelper()` should be used in test helper funcs. But here we are not in such helper func. Let me know if I miss something.

```suggestion
```